core/domain/fs_domain.py

```diff
@@ -107,34 +64,15 @@ def read(self):
         self._content = ''
         return content
 
-    @property
-    def metadata(self):
-        """Returns the file metadata model instance.
-
-        Returns:
-            FileMetadataModel. The file metadata model instance.
-        """
-        return self._metadata
-
-    @property
-    def version(self):
-        """Returns the version number of the file.
-
-        Returns:
-            int. The version number of the file.
-        """
-        return self._version
-
 
 class GeneralFileSystem(python_utils.OBJECT):
-    """The parent class which is inherited by both DatastoreBackedFileSystem
-    and GcsFileSystem as the member variables in both classes are the same.
+    """The parent class which is inherited by GcsFileSystem as the member
+    variables in both classes are the same.
```

Comment:
Drop the last clause ("as ..." onwards) -- I don't think it's needed.

Reply:
Done.